client/src/pages/CreateOrganisationProfile.js

```diff
@@ -368,6 +329,190 @@ const CreateOrgProfile = (props) => {
             <WhiteSpace />
             <WhiteSpace />
           </div>
+          <Flex
+            direction="row"
+            align="start"
+            justify="between"
+          >
+            <InputWrapper>
+              <InputGroup>
+                <Label
+                  style={styleLabel}
+                  label={
+                    <Trans i18nKey="profile.org.seeking">
+                      <b>
+                        request
+                      </b>
+                    </Trans>
+                  }
+                />
+                <Controller
```

Comment:
I think the name is required for these Controller components. It's throwing an error when I click on the Request check boxes. https://react-hook-form.com/api/#Controller.

Also, for UX, it might be better if the label is clickable too